config/build/dpkg_build/postinst.sh

```diff
@@ -171,6 +171,27 @@ if [ -d "${DLTCONNECTOR_PATH}" ]; then
    echo "sudo dpkg -i ${DLTCONNECTOR_PATH}${DLTCONNECTOR_NAME}"
 fi
 
+#
+# Remind user to install OpenJDK for Java
+#
+###############################################################################
```

Comment:
Hi Thong,
this is a usability issue:
Most of the users will not know genpackagedoc as this is only required if you contribute to our repositories and write documentation which is plantuml based.
Therefore: Please don't ask, just try to install JDK. If it fails even no error message shall be created, we just proceed installation.
Thank you,
 Thomas